app/components/video_controls.js

```diff
@@ -1,5 +1,4 @@
-// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
-// See License.txt for license information.
+// https://github.com/charliesbox/react-native-media-controls
```

Comment:
I'm not sure if we can use this. Their package.json lists the license as MIT, but it's not listed in their repository with the full license terms. Also, we don't have it included in the NOTICE.txt either. We should talk to the PMs to see if we can actually use this or if we have to roll our own

Reply:
Well this is actually a modified version of that library as that library as way too old and nothing was actually working